src/elements/content-explorer/__tests__/ContentExplorer-test.js

```diff
@@ -39,4 +39,11 @@ describe('elements/content-explorer/ContentExplorer', () => {
             expect(instance.fetchFolder).toHaveBeenCalledWith('123', false);
         });
     });
+
+    describe('render()', () => {
+        test('should render correctly with default props', () => {
+            const wrapper = getWrapper({});
+            expect(wrapper).toMatchSnapshot();
```

Comment:
Reposting my prior comment:

>Changes to snapshots at such a high level are often ignored, reducing their value. Ideally, tests should assert specific expectations and only use snapshots where they can't be avoided.

Reply:
Apologies, I thought your prior comment only pertained to the snapshot test with specific props. I'll take out this snapshot too. I'll also read up more on using jest and look deeper into our codebase to better understand how we test higher level components.